plugins/modules/idrac_diagnostics.py

```diff
@@ -683,6 +683,8 @@ def __export_diagnostics(self, payload):
         if not diagnostics_file_name:
             now = datetime.now()
             hostname = self.module.params.get('idrac_ip')
+            hostname = self.expand_ipv6(hostname)
```

Comment:
just make sure if this modifies the original input

Reply:
it will not modify.